india_forecast_app/models/all_models.yaml

```diff
@@ -0,0 +1,27 @@
+# Here we define all the models that are available in the app
```

Comment:
This is a bit of a nit but just to make this consistent should we have the same ordering for each of the model configs here, perhaps in the order they are defined in the pydantic model? I think the order there is name, type, id, version, client, asset type

Also client is not in the first two models, I know it is not strictly needed since there are defaults for this in the pydantic model but could be nice to include again just for consistency in the config

Reply:
Thanks for this, ive updated it, so its all the same order, and added client despite default. Makes it easier to read